Release 1.12.2 (#498)
heremaps/here-data-sdk-typescript: CHANGELOG.md

## v1.12.2 (05/01/2022)

**olp-sdk-dataservice-read**

> - Minor change in poll method (https://github.com/heremaps/here-data-sdk-typescript/pull/495).

Reviewer: ```suggestion
- `StreamLayerClient::poll()` now blocks sending a backend request if there are no messages to submit (https://github.com/heremaps/here-data-sdk-typescript/pull/495).
```

> - Not throw on sparse data layers (https://github.com/heremaps/here-data-sdk-typescript/pull/496).

Reviewer: ```suggestion
- `getTile` now does not throw an error on sparse versioned or volatile layers (https://github.com/heremaps/here-data-sdk-typescript/pull/496).
```

> - Improve Interactive Map Layers API (https://github.com/heremaps/here-data-sdk-typescript/pull/497).

Reviewer 2: ```suggestion
- Improved the interactive map layer API (https://github.com/heremaps/here-data-sdk-typescript/pull/497).
```
